app/lib/frontend/handlers/routes.dart

```diff
@@ -200,6 +200,10 @@ class PubSiteService {
   Future<Response> accountPackagesPage(Request request) async =>
       accountPackagesPageHandler(request);
 
+  @Route.get('/my-liked-packages')
```

Comment:
Maybe this could become a sub-page under `/my-packages`, e.g. `/my-packages/liked`?

Reply:
I think that's a mis-use... `/my-packages` is packages you own, `/liked` is not a subset of that.

Reply:
I agree if the menu becomes much bigger, but I think it is okay as long as we don't have more entries.

Reply:
fair point!